specify/specify7
# Allow having tree stats use direct rank names (and climb tree)

Allows defining a query like 'geography.Country'. Looks for name in the {TREE}treedefitem table. 

The previous code allowed just to modify the start values in the last query field, but these changes allow them to be generic. 

TO TEST:
1. Go to stats. All the previous versions of stats should still work - the query would be different in the query builder. New ranks will be visible in the add stats dialog (all ranks above Earth / Life are shown). 
2. The following query should be present (if rank is continent): 
![image](https://github.com/specify/specify7/assets/61122018/1425278f-32a1-43e5-97d7-3e6c771bb31a)


Change 2:
Removes distinct from locality stat
